scipy/integrate/_quadrature.py

```diff
@@ -522,16 +522,16 @@ def _basic_simpson(y, start, stop, x, dx, axis):
 
 # Note: alias kept for backwards compatibility. simps was renamed to simpson
 # because the former is a slur in colloquial English (see gh-12924).
-def simps(y, x=None, dx=1.0, axis=-1, even='avg'):
+def simps(y, x=None, dx=1.0, axis=-1, **kwds):
     """An alias of `simpson`.
 
     `simps` is kept for backwards compatibility. For new code, prefer
     `simpson` instead.
     """
-    return simpson(y, x=x, dx=dx, axis=axis, even=even)
+    return simpson(y, x=x, dx=dx, axis=axis, **kwds)
 
 
-def simpson(y, x=None, dx=1.0, axis=-1, even='avg'):
+def simpson(y, x=None, dx=1.0, axis=-1, **kwds):
```

Comment:
I think this will break existing code if they are using positional arguments? Instead, I think it would be easier to set the default to None and then later raise a deprecation warning if `even is not None`
```suggestion
def simpson(y, x=None, dx=1.0, axis=-1, even=None):
```